server/util/tracing/tracing.go

```diff
@@ -189,3 +193,20 @@ func ExtractProtoTraceMetadata(ctx context.Context, metadata *tpb.Metadata) cont
 	p := otel.GetTextMapPropagator()
 	return p.Extract(ctx, newTraceMetadataProtoCarrier(metadata, nil))
 }
+
+// StartSpan starts a new span named after the calling function.
+func StartSpan(ctx context.Context, opts ...trace.SpanOption) (context.Context, trace.Span) {
+	span := trace.SpanFromContext(ctx)
+	if !span.IsRecording() {
+		return ctx, span
+	}
+
+	rpc := make([]uintptr, 1)
+	n := runtime.Callers(2, rpc[:])
+	fn := ""
+	if n > 0 {
+		frame, _ := runtime.CallersFrames(rpc).Next()
+		fn = filepath.Base(frame.Function)
+	}
+	return otel.GetTracerProvider().Tracer("buildbuddy.io").Start(ctx, fn, opts...)
```

Comment:
nit: can we make "buildbuddy.io" a const in the tracing package?

Reply:
Done.